README.md

```diff
@@ -130,6 +125,27 @@ If you want to run cloud integration tests, ensure you log in to the cloud:
 
 This collection is tested using GitHub Actions. To know more about CI, refer to [CI.md](https://github.com/https://github.com/redhat-cop/cloud.aws_ops/blob/main/CI.md).
 
+## Contributing to this collection
+
+We welcome community contributions to this collection. If you find problems, please open an issue or create a PR against this collection repository.
+
+## Support
+
+You can also join us on:
+
+- Libera.Chat IRC - the ``#ansible-aws`` [irc.libera.chat](https://libera.chat/) channel
```

Comment:
I would suggest we remove this link and point people to the Ansible forum AWS group instead, since that's the recommended communication method. I would also move it to the Contributing section since that is not an official channel for making support requests.